app/view_models/v1/travel_and_waiting.rb

```diff
@@ -0,0 +1,46 @@
+module V1
+  class TravelAndWaiting < BaseViewModel
+    INCLUDED_TYPES = %w[travel waiting].freeze
+
+    attribute :work_items
+
+    def table_fields
+      work_item_data.map do |work_type, total_cost, total_time_spent|
+        [
+          work_type,
+          NumberTo.pounds(total_cost),
+          "#{total_time_spent}min",
+        ]
+      end
+    end
+
+    def total_cost
+      NumberTo.pounds(
+        work_item_data.sum { |_, total_cost, _| total_cost }
+      )
+    end
+
+    delegate :any?, to: :work_item_data
+
+    private
+
+    # rubocop:disable Metrics/CyclomaticComplexity
+    def work_item_data
+      @work_item_data ||=
+        work_items.map { |work_item| WorkItem.build_self(work_item) }
```

Comment:
The similar blocks found pointer above - is there some DRYing out we can do around this?